app/Main.hs

```diff
@@ -1,7 +1,11 @@
 module Main (main) where
 
-import Hintman (someFunc)
+import Network.Wai.Handler.Warp   (run)
+
+import Server
 
 
 main :: IO ()
-main = someFunc
+main = do
+    print ("Starting cookbook-generic at http://localhost:8000" :: Text)
+    run 8000 app
```

Comment:
I guess we can move it into the library under `Hintman.App` module, can name it like `runHintman`, or smth like this. So we will keep this module minimal and have all logic in the library itself.

Reply:
ok